File: app/src/main/java/ru/artempugachev/popularmovies/data/VideoResponse.java
package ru.artempugachev.popularmovies.data;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Data class for video response
 */

public class VideoResponse {
    @SerializedName("id")
    private String id;


    @SerializedName("results")
    private List<Video> results;

    public List<Video> getResults() {
        return results;
    }
}

File: app/src/main/java/ru/artempugachev/popularmovies/moviedetails/api/VideoResponse.kt
package ru.artempugachev.popularmovies.moviedetails.api

import com.google.gson.annotations.SerializedName

import ru.artempugachev.popularmovies.moviedetails.api.Video

/**
 * Data class for video response
 */

data class VideoResponse (
    @SerializedName("id")
    val id: String,

    @SerializedName("results")
    val results: List<Video>
)
